src/lib/hooks/hasConnectionChange.js

```diff
@@ -134,22 +127,19 @@ export const useConnectionGun = () => {
   const gunClose = useCallback(() => {
     log.debug('gun close')
     isGunConnection()
-  }, [])
+  }, [isGunConnection])
```

Comment:
About deps in useMemo / useEffect / useCallback
There is react-hooks/exhaustive-deps rule could help to not forgot to specify deps

We need to:
1. make sure we have react hooks plugin version including this rule
2. enable it in .eslintrc
3. Re-run lint for all app, fix issues found

Shall i to add a separate task (in icebox i think) or you'll do it ?

Reply:
please install the pluigin and enable in eslint